.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 20.19.2
```

Comment:
I thought about adding this to the gitignore but it could actually be helpful to people using asdf for their version manager and I don't think it hurts anyone not using asdf.